File: games/bounce-evasion/match-generator/core/src/com/planet_lia/match_generator/libs/MatchDetails.java
package com.planet_lia.match_generator.libs;

/**
 * Match details data sent to each bot
 */
public class MatchDetails {
    public BotDetails[] botsDetails;

    // Index of the bot in botsDetails to which
    // this instance of MatchDetails was sent to
    public int yourBotIndex;

    public MatchDetails(BotDetails[] botsDetails, int yourBotIndex) {
        this.botsDetails = botsDetails;
        this.yourBotIndex = yourBotIndex;
    }
}

File: games/lia-1/bots/kotlin/src/core/MatchDetails.kt
package core

class MatchDetails(
        /**
         * Index of the bot in botsDetails to which
         * this instance of MatchDetails was sent to
         */
        var yourBotIndex: Int,

        /**
         * List of all bots that participate in this
         * match with their details
         */
        var botsDetails: Array<BotDetails>
)
